platforms/posix/CMakeLists.txt

```diff
@@ -5,8 +5,6 @@ include_directories(${CMAKE_CURRENT_BINARY_DIR})
 
 get_property(module_libraries GLOBAL PROPERTY PX4_MODULE_LIBRARIES)
 
-set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin)
```

Comment:
This changes the default CMAKE_RUNTIME_OUTPUT_DIR from   CMAKE_CURRENT_BINARY_DIR  (./platforms/posix) to (./bin).